.ci/jobs/e2e-tests-new.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- job:
```

Comment:
Maybe it make sense to update existed job instead of creating new job? Why do we need to create a new job?

Reply:
I'd like to run this in parallel to the current one until we verify it works correctly and only then update the current job. Otherwise we might lose e2e coverage.